tests/login.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '@playwright/test';
+import { LoginPage } from '../pages/loginPage';
+import { SecureAreaPage } from '../pages/secureAreaPage';
+
+test('Successfully login with the correct credentials', async ({ page }) => {
+    let loginPage = new LoginPage(page);
+    await loginPage.navigateToPage('https://the-internet.herokuapp.com/login');
+    await loginPage.enterUsername('tomsmith');
+    await loginPage.enterPassword('SuperSecretPassword!');
+    await loginPage.clickLoginButton();
```

Comment:
this block of code is re-used multiple times. try to create a reusable function to avoid this much redundancy in the code